shapely/predicates.py

```diff
@@ -862,10 +862,11 @@ def intersects(a, b, **kwargs):
 def overlaps(a, b, **kwargs):
     """Return True if A and B spatially overlap.
 
-    A and B overlap if they have some but not all points in common, have the
-    same dimension, and the intersection of the interiors of the two geometries
-    has the same dimension as the geometries themselves.  That is, only polyons
-    can overlap other polygons and only lines can overlap other lines.
+    A and B overlap if they have some but not all points/space in common,
+    have the same dimension, and the intersection of the interiors of the two geometries
+    has the same dimension as the geometries themselves. That is, only polyons
+    can overlap other polygons and only lines can overlap other lines. If A covers or is
+    within B, overlaps won't be True.
```

Comment:
@theroggy can you format this block to 72 chars width? It's an archaic practice that I followed when I wrote the package and it really does help docs fit in the terminal when someone does a `help(overlaps)`.

Ideally we should set up a pre-commit hook for this 🤔

Reply:
Sure. Note that there are very few docstrings now that seem to be limited to 72 characters at first sight. Most, not all seem to be restricted to 80 characters though.

Haven't used it yet, but ruff seems to supports having a different line length for "comments and docstring" lines:
https://docs.astral.sh/ruff/rules/doc-line-too-long/.